app/app.go

```diff
@@ -173,6 +179,41 @@ var (
 	// DefaultNodeHome default home directories for the application daemon
 	DefaultNodeHome string
 
+	// ModuleBasics defines the module BasicManager is in charge of setting up basic,
+	// non-dependant module elements, such as codec registration
+	// and genesis verification.
+	ModuleBasics = module.NewBasicManager(
+		auth.AppModuleBasic{},
+		genutil.NewAppModuleBasic(genutiltypes.DefaultMessageValidator),
+		bank.AppModule{},
+		capability.AppModuleBasic{},
+		staking.AppModule{},
+		mint.AppModule{},
+		distr.AppModuleBasic{},
+		gov.AppModule{AppModuleBasic: gov.NewAppModuleBasic(getGovProposalHandlers())},
+		params.AppModuleBasic{},
+		crisis.AppModule{},
+		slashing.AppModule{},
+		feegrantmodule.AppModuleBasic{},
+		upgrade.AppModuleBasic{},
+		evidence.AppModuleBasic{},
+		vesting.AppModuleBasic{},
+		// schedulermodule.AppModuleBasic{},
+		// consensusmodule.AppModuleBasic{},
+		// valsetmodule.AppModuleBasic{},
+		wasm.AppModuleBasic{},
+		evm.AppModuleBasic{},
+		gravitymodule.AppModuleBasic{},
+		// palomamodule.AppModuleBasic{},
+		// treasurymodule.AppModuleBasic{},
+		// consensus.AppModuleBasic{},
+		transfer.AppModuleBasic{},
+		ibc.AppModuleBasic{},
+		tendermint.AppModuleBasic{},
```

Comment:
Why we need this module?

Reply:
Replaced `tendermint.AppModuleBasic{}` with `ibtctm.AppModuleBasic{}`  and pushed